components/brave_shields/browser/domain_block_navigation_throttle.cc

```diff
@@ -122,6 +126,11 @@ DomainBlockNavigationThrottle::WillStartRequest() {
   if (tab_storage->IsProceeding())
     return content::NavigationThrottle::PROCEED;
 
+  // If this URL is about to be debounced, don't show an interstitial.
+  GURL debounced_url;
+  if (debounce_service_->Debounce(request_url, &debounced_url))
```

Comment:
it will crash if `kBraveDebounce` is disabled.

Reply:
fixed in d4cbe8b600